src/Makefile.am

```diff
@@ -18,7 +20,7 @@ libasr_la_CPPFLAGS =	-I$(top_srcdir)/src -I$(top_srcdir)/openbsd-compat
 
 libasr_la_CFLAGS=	-DASR_OPT_THREADSAFE=0
 
-libasr_la_LDFLAGS=	-version-info 0:2:0
+libasr_la_LDFLAGS=	-version-info 0:2:0 -lcrypto
```

Comment:
I'm fairly certain that this needs to be changed but I'm not unsure what changes are needed.